### PR TITLE
new module rtgtools/rocplot

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by nvnieuwk** [comment]
Closes #2744

File: tests/modules/nf-core/rtgtools/rocplot/main.nf [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    ]
+
+    compressed_sdf = [
+        [ id:'test' ],
+        file(params.test_data['homo_sapiens']['genome']['genome_21_sdf'])
+    ]
+
+    sdf = UNTAR( compressed_sdf ).untar
+
+    RTGTOOLS_VCFEVAL ( input, sdf )
```
Comment (pinin4fjords) [inline tests/modules/nf-core/rtgtools/rocplot/main.nf:29]: Shouldn't there be an output of this process in the test data repo so it doesn't have to keep regenerating in CI? Or is it really trivial?

File: modules/nf-core/rtgtools/rocplot/meta.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  - rocplot
+  - validation
+  - vcf
+tools:
+  - "rtgtools":
+      description: "RealTimeGenomics Tools -- Utilities for accurate VCF comparison and manipulation"
+      homepage: "https://www.realtimegenomics.com/products/rtg-tools"
+      documentation: "https://github.com/RealTimeGenomics/rtg-tools"
+      tool_dev_url: "https://github.com/RealTimeGenomics/rtg-tools"
+      doi: ""
```
[the remark this answers is not in the document]
Comment (nvnieuwk) [reply modules/nf-core/rtgtools/rocplot/meta.yml:14]: What do you mean?

File: tests/modules/nf-core/rtgtools/rocplot/main.nf (hunk 1 above)
Comment (nvnieuwk) [reply tests/modules/nf-core/rtgtools/rocplot/main.nf:29]: It's a very low footprint process, but I can add it if you would prefer it :)

File: modules/nf-core/rtgtools/rocplot/meta.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (pinin4fjords) [reply modules/nf-core/rtgtools/rocplot/meta.yml:14]: I mean if there's no DOI, it's probably not that useful to have an empty value here.

File: tests/modules/nf-core/rtgtools/rocplot/main.nf (hunk 1 above)
Comment (pinin4fjords) [reply tests/modules/nf-core/rtgtools/rocplot/main.nf:29]: Other views may differ, but whenever I've done this I've regretted it an ended up putting the test data in.

File: modules/nf-core/rtgtools/rocplot/meta.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (nvnieuwk) [reply modules/nf-core/rtgtools/rocplot/meta.yml:14]: Oh true! I'll remove it

File: tests/modules/nf-core/rtgtools/rocplot/main.nf (hunk 1 above)
Comment (pinin4fjords) [reply tests/modules/nf-core/rtgtools/rocplot/main.nf:28]: Just that I ended up using the inputs for other things, waiting for the upstream tasks to run in CI etc. I won't fight you on it though :-D

**Comment by nvnieuwk** [comment]
> Looks okay to me, but I don't know what's going on with the conda

Me neither... but it should be fine I think (at least docker and singularity are running good :partying_face:  )

**Pull request merged**